src/main/java/com/example/demo/matcher/models/Order.java

```diff
@@ -1,21 +1,35 @@
 package com.example.demo.matcher.models;
 
+import lombok.Getter;
+import lombok.ToString;
+
 import java.time.LocalDateTime;
+import java.util.UUID;
 
+@Getter @ToString
 public class Order {
-    String id;
-    String account;
-    float price;
-    float quantity;
-    OrderAction action;
-    LocalDateTime datetime;
+    final UUID id;
+    final String accountId;
```

Comment:
What's the difference between `id` and `accountId`?

Reply:
id is the id for the order, and accountId is the id for the account. Should I rename it to orderId?

Really accountID should also be a UUID type - I'll get on that.

Reply:
done